src/Compilers/CSharp/Portable/Symbols/Synthesized/GeneratedNameKind.cs

```diff
@@ -38,6 +38,8 @@ internal enum GeneratedNameKind
         AsyncBuilderField = 't',
         DynamicCallSiteContainerType = 'o',
         DynamicCallSiteField = 'p',
+        AsyncIteratorPromiseOfValueOrEndBackingField = 'v',
```

Comment:
unrelated? why is this fil;e unordered? it is very confusing to read/reason about.

Reply:
I agree. Finding the next available letter was not very easy.
I'd re-order it, but I don't really understand the groups (especially what does "Currently not parsed" mean).
I'll leave as-is.

Reply:
"Currently not parsed" means not handled in `GeneratedNames.TryParseGeneratedName`.

---
In reply to: [204532707](https://github.com/dotnet/roslyn/pull/28218#discussion_r204532707) [](ancestors = 204532707)